it/src/test/scala/quasar/fs/WriteFilesSpec.scala

```diff
@@ -24,6 +24,18 @@ class WriteFilesSpec extends FileSystemTest[FileSystem](FileSystemTest.allFsUT)
     "Writing Files" should {
       step(deleteForWriting(run).runVoid)
 
+      "opening a file should create it" >>* {
```

Comment:
It is a bit of syntactic sugar defined in `FileSystemTest` that allows the body to return a value of `Free[S, A: AsResult]` which is then automatically interpreted using the current interpreter, rather than having to explicitly run it. Just allows us to reduce a bit of noise at times.